orchestrator/build.gradle

```diff
@@ -97,6 +98,18 @@ task fatJar(type: Jar) {
     archiveName = "${baseName}.${extension}"
 }
 
+task createDeb(type: Deb) {
+    packageName = "datahelix"
+    packageDescription = "The open-source data generator."
```

Comment:
That's rather a bold claim!!

Reply:
Nabbed it from the description on [chocolatey](https://chocolatey.org/packages/datahelix/2.1.10) for consistency, not too attached to it if anyone has a better suggestion though!